src/provider/provider_file_memory.c

```diff
@@ -545,6 +553,14 @@ static umf_result_t file_get_ipc_handle_size(void *provider, size_t *size) {
         return UMF_RESULT_ERROR_INVALID_ARGUMENT;
     }
 
+    file_memory_provider_t *file_provider = (file_memory_provider_t *)provider;
+    if (!file_provider->IPC_enabled) {
+        /// IPC is enabled only for UMF_MEM_MAP_SHARED or UMF_MEM_MAP_SYNC visibility
```

Comment:
I guess these comments are redundant, they say pretty much the same as the LOG message (applies to all these LOGs)

Reply:
Done